modules/installation-supported-aws-machine-types.adoc

```diff
@@ -408,6 +408,11 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 |===
 ====
 
+[IMPORTANT]
+====
+`C7g` instance types are only available on AWS Graviton3 processors.
```

Comment:
Not quite sure of the point of this comment.  IIUC, by definition, each instance class is provided on a single CPU type.  If this is meant to indicate limited availability (as above), then let's make that explicit?

Reply:
Gotcha! That makes sense, I think I'll remove the statement in that case. Thank you!